admin/src/components/Input/CKEditor/plugins/StrapiUploadAdapter.js

```diff
@@ -140,21 +139,23 @@ class Adapter {
         );
       }
 
-      
       const { backendUrl, responsive } = this.options || {};
-      
-      if (response[0].formats && responsive) {
-        const { name, url, alternativeText, formats } = response[0];
-        let urls = { default: backendUrl + url };
-        let keys = Object.keys(formats).sort((a, b) => formats[a].width - formats[b].width);
+      const { name, url, alternativeText, formats, provider } = response[0];
+      const defaultUrl = provider !== "local" ? url : backendUrl + url;
+
+      if (formats && responsive) {
+        let urls = { default: defaultUrl };
+        let keys = Object.keys(formats).sort(
+          (a, b) => formats[a].width - formats[b].width
+        );
         keys.map((k) => (urls[formats[k].width] = backendUrl + formats[k].url));
```

Comment:
Isn't there a mistake here too? I'm having the same problem now for the src set. 

srcset="http://localhost:1337https://SPACE_PATH/870fd27ba154892bc07c9164bbbab4a2.png 64w"